summarizer/model_service/builds/base/Containerfile

```diff
@@ -0,0 +1,14 @@
+# Pass build args to build a specified model from a specified local directory
+FROM registry.access.redhat.com/ubi9/python-39:latest
+WORKDIR /locallm
+ARG MODEL_FILE=llama-2-7b-chat.Q5_K_S.gguf
+ARG MODEL_LOCAL_DIR=models
+COPY $MODEL_LOCAL_DIR/$MODEL_FILE /locallm/models/
```

Comment:
@MichaelClifford will these ARGS work? I thought we needed to export just the MODEL_FILE env var so kept the dir/file separate args - but maybe that's not required.